### PR TITLE
Removing list syntax from CI badges

### PR DESCRIPTION
**Pull request by mbbroberg**
List view renders incredibly small and badges stack quite nicely on top of each other. This may have changed when GitHub updated how they render markdown, so its a good time to adjust here.

### CODE REVIEW

**Comment by mbbroberg** [comment]
Note that I've tested this PR on https://github.com/intelsdi-x/snap/pull/1664

File: lib/pluginsync/util.rb [hunk 1]
```diff
@@ -139,8 +140,8 @@ def self.html_list(list)
       case list
       when ::Array
         if list.size > 1
-          items = list.collect{|i| "<li>#{i}</li>"}.join('')
-          "<ul>#{items}</ul>"
+          items = list.collect{|i| "#{i}"}.join(' ')
+          "#{items}"
```
Comment (nanliu) [inline lib/pluginsync/util.rb]: You can omit this line if you don't need to wrap everything around <ul> <li>. In fact we don't even need the temp variable items or collect. These two lines can simply be:
```ruby
list.join(' ')
```

Comment (mbbroberg) [inline lib/pluginsync/util.rb]: Oh gotcha. Given that, I think I can remove the `case` and make it simpler: 

```ruby
def self.html_list(list)
      list.join(' ') if list.is_a?(Array)
end
```

File: lib/pluginsync/util.rb [hunk 2]
```diff
@@ -137,17 +137,7 @@ def self.plugin_capitalize(word)
...
-        if list.size > 1
-          items = list.collect{|i| "#{i}"}.join(' ')
-          "#{items}"
-        else
-          "#{list.first}"
-        end
-      else
-        list
-      end
+      list.join(' ') if list.is_a?(Array)
```
Comment (nanliu) [inline lib/pluginsync/util.rb]: This lost the default, also ::Array != Array.
```ruby
if list.is_a? ::Array 
  list.join(' ')
else
 list
end
```

Comment (mbbroberg) [inline lib/pluginsync/util.rb]: I'll pick your brain on how after this @nanliu 😄

Comment (mbbroberg) [inline lib/pluginsync/util.rb]: @nanliu rebased into the last commit.

**Comment by nanliu** [comment]
Squash and merge

**Pull request merged**